src/sentry/api/serializers/base.py

```diff
@@ -1,15 +1,19 @@
-from typing import Any, Mapping, Optional, Sequence, Union
+from typing import Any, Callable, List, Mapping, MutableMapping, Optional, Sequence, Union
 
 import sentry_sdk
 from django.contrib.auth.models import AnonymousUser
 
+from sentry.utils.json import JSONData
+
+K = Any
```

Comment:
I think this is not a [typevar](https://mypy.readthedocs.io/en/stable/generics.html#generic-functions), so it's the same as `Callable[[Any], Any]`, meaning that a decorated class will lose its type once the decorator is applied in either case.

```
K = TypeVar("K")

Reply:
I see a commit that introduced it but seems like it got reverted

Reply:
Yes, please either convert this to a proper TypeVar or remove this alias.